flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/schema/Selectors.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.cdc.connectors.mysql.schema;
+
+import org.apache.flink.cdc.common.utils.Predicates;
+
+import io.debezium.relational.TableId;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+
+/** Selectors for filtering tables. */
+public class Selectors {
```

Comment:
Why do we need to add this class ?

Reply:
Because we need to translate tables.exclude to a filter. We can not get it by dbzMySqlConfig. 
```suggestion
public class Selectors {z
```

Reply:
Because we need to translate tables.exclude to a filter. We can not get it by dbzMySqlConfig. getTableFilters which need the whole debezium config.
I alse want to use org.apache.flink.cdc.common.schema.Selectors, but  it is not so good:
1. the param is org.apache.flink.cdc.common.event.TableId rather than io.debezium.relational.TableId, so I can use it directly in mysql cdc source.
2. org.apache.flink.cdc.common.schema.Selectors.SelectorsBuilder#includeTables will parse "databaseName.tableName" as <null, databaseName, tableName>, the org.apache.flink.cdc.connectors.mysql.source.utils.TableDiscoveryUtils#listTables will return <databaseName, null, tableName>. Thus, the selector cannot filter table from mysql cdc.